src/sc/Deserialize.js

```diff
@@ -0,0 +1,108 @@
+import StackItemType from './StackItemType'
+import { reverseHex } from './../utils'
+
+const Deserialize = serializedArray => {
+  // Split into bytes of 2 characters
+  const byteArray = serializedArray.match(/.{2}/g)
+
+  // Initialize under root byte
+  let iterator = -1
+  let iteratedByte
+
+  const setIteratedByte = () => {
+    iterator++
+    iteratedByte = byteArray[iterator]
+    return iteratedByte
+  }
```

Comment:
Any special reason why we want to have the iterator set a global var instead of just using the return value? I see some places where you use the global and some where you use the return.

Reply:
Yeah I used 'm both but I can refactor them 😄